### PR TITLE
convert addGroup to UserGroupManager

### PR DESCRIPTION
**Pull request by Universal-Omega**

### CODE REVIEW

File: maintenance/removePII.php [hunk 1]
```diff
@@ -636,23 +636,23 @@ public function execute() {
 		);
 
 		$user = User::newSystemUser( 'MediaWiki default', [ 'steal' => true ] );
+
+		$UserGroupManager = MediaWikiServices::getInstance()->getUserGroupManager();
```
Comment (paladox) [inline maintenance/removePII.php:640]: Should use coding convention e.g. camelCasing.

Comment (Universal-Omega) [reply maintenance/removePII.php:640]: oh, right, thanks!

Comment (Universal-Omega) [reply maintenance/removePII.php:640]: fixed

Comment (paladox) [reply maintenance/removePII.php:640]: doesn't seem fixed to me, still showing upper case.

Comment (Universal-Omega) [reply maintenance/removePII.php]: ok, now it's fixed, sorry

**Pull request merged**